pkg/shares/share_sequence.go

```diff
@@ -62,6 +70,17 @@ func (s ShareSequence) validSequenceLen() error {
 	return nil
 }
 
+func (s ShareSequence) isPadding() (bool, error) {
+	if len(s.Shares) != 1 {
```

Comment:
Is this not supposed to be. Can't a padding sequence be multiple shares? 
```suggestion
	if len(s.Shares) > 0 {
```

Reply:
A padding sequence can only be one share. 

Since there are no bits in the universal share prefix that indicate if a share is namespace padding, we abuse two existing fields to determine if a share is namespace padding. A namespace padding share has:
1. a sequence start indicator of `1` 
2. a sequenceLen of `0`

Reply:
> A padding sequence can only be one share.

So tail padding is not considered a single sequence even although they would all have the same namespace

Reply:
Yep exactly.

Reply:
Ok my (admittedly internal) definition of a sequence was a set of shares that all had the same namespace

Reply:
Yea it's definitely a bit confusing and that what you describe is the original idea for a sequence.

TBH I think padding sequences all being of length one is a hack that we should reconsider. I'm in favor of using some other signaling mechanism in the universal share prefix to identify namespace padding shares or padding shares in general.